internal/cache/controller_obj_cache.go

```diff
@@ -1382,7 +1417,9 @@ func (c *AviObjCache) PopulateSSLKeyToCache(client *clients.AviClient, cloud str
 				cacert = ca.(string)
 			}
 		}
+		//TODO: Check with team : What is reason of recomputing checksum here again.
 		SslKeyData[i].CloudConfigCksum = lib.SSLKeyCertChecksum(SslKeyCacheObj.Name, SslKeyCacheObj.Cert, cacert)
```

Comment:
@sudswasavi @monotosh-avi : What is reason of computing sslkeydata checksum here again.  We are doing it at AviPopulateAllSSLKeys() call? Shallow-cpy is reason?

Reply:
To me, looks like we don't need this.